recipes/perl-vcftools-vcf/build.sh

```diff
@@ -1,12 +1,9 @@
 #!/bin/bash
 set -eu
-
 mkdir -p perl-lib/lib
 cp src/perl/*.pm perl-lib/lib
-# Fix version to something cpan installable tools like
-sed -i.bak 's/r953/0.953/' perl-lib/lib/Vcf.pm
 # Fix naming scheme of package to make Builder happy
 sed -i.bak 's/Vcf.pm.  Module/Vcf - Module/' perl-lib/lib/Vcf.pm
 cp $RECIPE_DIR/Build.PL perl-lib
 cd perl-lib
-cpanm -v -i .
```

Comment:
Why can you not call Build.PL directly? https://github.com/bioconda/bioconda-recipes/pull/4798/files#diff-07b0d6ce7a6e79394d4384e7db728326R4

My fear is that cpanm does more magic things, like pulling down other libs that should be defined more explicitly.

Reply:
Indeed it works perfectly well thanks ! I have removed the cpan dependency accordingly.